packages/blade/src/components/Card/Card.tsx

```diff
@@ -83,12 +83,21 @@ export type CardProps = {
    * Sets the height of the card
    */
   height?: BoxProps['height'];
+    /**
+    Sets minimum height of the card
+   */
+  minHeight?: BoxProps['minHeight'];
+      /**
+    Sets minimum width of the card
+   */
```

Comment:
Please format the code same as how `width` is formatted. 

```suggestion
  /**
   * Sets minimum width of the card
   */
```

Reply:
Ok I'll do that and apologies for not doing the formatting properly this time !